addon/components/materialize-checkbox.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-checkbox';
+
+export default Ember.Component.extend({
+  layout: layout,
+  tagName: 'p',
```

Comment:
Should not specify tag name since checkboxes can be contained in any element.

Users can specify the surrounding tag themselves or override when declaring  the component `{{materialize-checkbox tagName="p"}}`